src/shared/discoverQuery.ts

```diff
@@ -74,3 +74,9 @@ export const descriptionTransform = (description: string): string =>
   )
     // remove empty lines
     .replace(/\n{2,}/gm, '\n');
+
+export default {
```

Comment:
is this needed?  Did it not work to just import the functions?

Reply:
What I've found is that if you intend to stub something it needs to be imported the same way by both the source file and the test file. On top of that, sinon needs to have a module to stub instead of the function itself.

That means that sinon needs `import things from 'discoveryQuery`, which means that the source file has to do the same.

Reply:
eww.  That's going to be dangerous.  Put that on your list of "things you need to know about ESM"

Reply:
I also don't like that "mockability for tests" is driving the code structure.  Maybe something like esmock could let tests handle this kind of thing?

https://github.com/iambumblehead/esmock/wiki

and also help with all the `node:foo` stuff?